pubsub/kubemq/kubemq_events.go

```diff
@@ -0,0 +1,169 @@
+package kubemq
+
+import (
+	"context"
+	"github.com/dapr/components-contrib/pubsub"
+	"github.com/dapr/kit/logger"
+	"github.com/kubemq-io/kubemq-go"
+	"github.com/kubemq-io/kubemq-go/pkg/uuid"
```

Comment:
Please use `google/uuid` which is what we use throughout Dapr. This package also has methods that return errors (`NewRandom()`) rather than panic in case of errors.

Reply:
Changed as requested